### ISSUE TITLE
Enable option for DEV base path if configured from the package

### ISSUE THREAD

**Issue by ValerioGiuffrida** [WFP-VAM/DataBridgesAPI#9]
https://github.com/WFP-VAM/DataBridgesAPI/blob/a327b06cca1dd250bbcd8371faa33d18fd45c8de/data_bridges_client/configuration.py#L116

**Comment by paololucchino** [comment]
Hi @ValerioGiuffrida,

Unfortunately, the default `.base_path` URL is populated automatically from the API swagger.json. Even if one manually changes the path here, it would not change all the functions that map to the methods available on the endpoints, so it would not work.

If we want to have a client that exposes the DEV endpoints, it would need to be a separate package, build on the code that was generated from the dev swagger.json. we could call it `data-bridges-dev` or something like that. It really depends how long new methods sit in dev before they go to prod.

Basically `1 swagger -> 1 client`

**Comment by ValerioGiuffrida** [comment]
Will work to implement this as part of the API parameters then. 
It's more a matter of having a testing environment. No need for further modifications then!
